Symfony/CS/Fixer/Contrib/EchoToPrintFixer.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Sullivan Senechal <soullivaneuh@gmail.com>
+ */
+final class EchoToPrintFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        $echoTokens = $tokens->findGivenKind(T_ECHO);
+
+        foreach ($echoTokens as $echoIndex => $echoToken) {
+            $nextTokenIndex = $tokens->getNextMeaningfulToken($echoIndex);
+            $endTokenIndex = $tokens->getNextTokenOfKind($echoIndex, array(';'));
+            $canBeConverted = true;
+            for ($i = $nextTokenIndex; $i < $endTokenIndex; ++$i) {
+                if ($tokens[$i]->equals(',')) {
+                    $canBeConverted = false;
+                    break;
+                }
+            }
+
+            if (false === $canBeConverted) {
+                continue;
+            }
+
+            $tokens[$echoIndex]->override(array(T_PRINT, 'print'));
```

Comment:
(On 1.x there is no difference, but on 2.x there is: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/Symfony/CS/Tokenizer/Tokens.php#L852-L856)